static/js/dismiss_banner.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function() {
+  function setCookie(name, value, days) {
+    let expires = "";
+    let date = new Date(); // Create a new Date object
+
+    if (days) {
+      date.setTime(date.getTime() + days * 86400000); // Modify the existing Date object

```

Comment:
(nit)
```suggestion
      date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000); // Modify the existing Date object
```
makes the derivation a bit more obvious. Or define a constant for `days * 24 * 60 * 60 * 1000`.